cmd/web/tmpl/showpaste.tmpl

```diff
@@ -1,11 +1,18 @@
 {{template "header" .}}
 
+<link href="/static/css/prism.css" rel="stylesheet" />
+
 <div class="right">{{.PasterDisplayName}} <img style="width:32px;height:32px" src="{{.PasterProfilePicURL}}" /></div>
 
 <p>Created at {{.CreatedAt}}</p>
 
-<pre><code>{{.Data}}
+<div style="display:inline-block;">
+<pre><code class="{{.CSSClass}}">{{.Data}}
 </code></pre>
+</div>
 
 <a href="/paste/{{.ID}}">Permalink</a> - <a href="/paste/{{.ID}}/dl">Download</a> - <a href="/paste/{{.ID}}/raw">Raw</a>{{if eq .UserID .PasterUserID}} - <a href="/api/delete/{{.ID}}">Delete</a>{{end}}
+
+<script src="/static/js/prism.js" async></script>
```

Comment:
I think the pop in effect is going to happen regardless, the fundamental way that this library works is that it renders things on the page at runtime. Moving it to the top of the page and removing the async tag means that it will not pop in, but it makes the page load slightly slower.

 Which trade off do you think is better?

Reply:
Given how prism affects the UX, the page load feels slow, even if it's not technically slow.

I think we can improve perception by changing the default styles for the `<code>` section to match that of prism, such that when prism activates, it appears as if it's (mostly) the colors that pop in.